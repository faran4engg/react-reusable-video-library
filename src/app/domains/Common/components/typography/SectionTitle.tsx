import { FC } from 'react';
import { SectionTitleProps } from './types';

const SectionTitle: FC<SectionTitleProps> = ({ children }) => (
  <h2 className="my-2 text-lg font-semibold text-gray-500 dark:text-gray-300">
    {children}
  </h2>
);

export default SectionTitle;
