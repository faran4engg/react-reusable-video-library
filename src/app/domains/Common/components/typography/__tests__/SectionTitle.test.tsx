import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import SectionTitle from '../SectionTitle';

const renderer = createRenderer();

describe('<SectionTitle />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<SectionTitle>This is Section Title</SectionTitle>);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
