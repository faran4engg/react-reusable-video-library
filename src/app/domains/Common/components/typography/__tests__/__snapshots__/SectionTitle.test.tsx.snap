// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionTitle /> should render and match the snapshot 1`] = `
<h2
  className="my-2 text-lg font-semibold text-gray-500 dark:text-gray-300"
>
  This is Section Title
</h2>
`;
